connected to server at port 3000, connected the server to mongoDB and…

Shumba250 opened the pull request:
… tested GET, POST,PATCH, DELETE  endpoints and added create single post, read all posts, update the blog and delete a post

Shumba250 committed:
connected to server at port 3000, connected the server to mongoDB and tested GET, POST,PATCH, DELETE  endpoints and added create single post, read all posts, update the blog and delete a post

wakaflorien commented on node_mongo_tutorial/routes/router.js:
```diff
@@ -0,0 +1,54 @@
+
+router.get("/", async (req, res) => {
+	const blogs = await Article.find();
+	res.json(blogs);
```
would you re-format your responses and be consistent on it all over. 
eg: {status, message, data}

wakaflorien commented on node_mongo_tutorial/routes/router.js:
```diff
@@ -0,0 +1,54 @@
+		const result = await blog.save();
+		res.status(200).json(result);
+	} catch {
+		res.status(404);
```
no need to send multiple responses. you can combine them in one.

wakaflorien commented on node_mongo_tutorial/routes/router.js:
```diff
@@ -0,0 +1,54 @@
+		res.status(204).send();
+	} catch {
+		res.status(404);
+		res.send({ error: "Post doesn't exist!" });
```
i suggest you use single response.

wakaflorien commented on node_mongo_tutorial/routes/router.js:
```diff
@@ -0,0 +1,54 @@
+			blog.image = req.body.image;
+		}
+		const result = await blog.save();
+		res.status(200).json(result);
```
use this response convention on all responses and format json as i mentioned.

wakaflorien commented:
> 

work on the PR format too. follow the standard one. and that goes to the commits too. @Shumba250

Shumba250 committed:
feat(Articles): CRUD of Articles/blogs

created GET API that retireves all blogs from database
created GET API that retrieves a single blog from database
created PATCH API that updates a single blog
created DELETE API that deleted a single blog

finishes CRUD OF THE BLOGS

Shumba250 committed:
bug{blogs): CRUD of blogs

created GET API that retrieves the number of blogs in the database
used JSEND response syntax

wakaflorien commented:
> 

### PR can be improved follow the guide.
